charts/linkerd-crds/Chart.yaml

```diff
@@ -14,7 +14,7 @@ dependencies:
 - name: partials
   version: 0.1.0
   repository: file://../partials
-version: 1.8.0
+version: 1.8.1-edge
```

Comment:
Needs to be `1.9.0-edge` instead, so that if there's change in CRDs somewhere in 2.14.x, it can be bumped in that branch without appearing after this edge.

Reply:
Hmm, I see.